flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -502,9 +502,9 @@ protected void beforeInvoke() throws Exception {
 			// both the following operations are protected by the lock
```

Comment:
Could you please fix commit msg and PR title ("before")?

Reply:
Done)